plzcrawler/plzCrawler.py

```diff
@@ -23,53 +23,63 @@ def __init__(self, excelTableNameString, workSheetString, tableDirectoryRawStrin
         self.placeIdListe = []
 
     def openAddressTable(self):
-        """nimmt Name einer Exceltabelle in Form eines Strings,
-        den Namen des entsprechenden Tabellenblattes in Form eines Strings,
-        & die Windows-Tabellen-Directory in Form eines Raw-Strings;
-        navigiert zum angegebenen Windows-Pfad & öffnet die Tabelle als pandas-dataframe;
-        speichert Tabelle als pandas dataframe-object"""
-
-        # zu directory navigieren, die Tabelle enthält
+        # """nimmt Name einer Exceltabelle in Form eines Strings,

```

Comment:
Could you please decide for either the German version or the english one.
In this case English seems to be the way to go.

Reply:
Oh sorry ill work on it tommorow morning 🌞

Reply:
ill go with eng versioon